commit e12dc3278cb2f818e47a553832c6fb017d27a699
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Jul 29 18:00:03 2021 -0300

    Correções de matriz de confusão -- Maria Clara
    
    Coisas que ficaram pra trás em: Advmod x case, advmod x cc, compound x nmod e compound x amod.

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 3021d77..5bcdaf8 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -6734,9 +6734,9 @@
 34	forte	forte	ADJ	_	Gender=Masc|Number=Sing	35	amod	O	_
 35	controle	controle	NOUN	_	Gender=Masc|Number=Sing	32	obj	O	_
 36	geomorfológico	geomorfológico	ADJ	_	Gender=Masc|Number=Sing	35	amod	O	_
-37	como	como	ADV	_	_	39	cc	O	_
+37	como	como	SCONJ	_	_	39	mark	O	_
 38	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	39	cop	O	_
-39	possível	possível	ADJ	_	Gender=Masc|Number=Sing	32	conj	O	_
+39	possível	possível	ADJ	_	Gender=Masc|Number=Sing	32	advcl	O	_
 40	visualizar	visualizar	VERB	_	VerbForm=Inf	39	csubj	O	_
 41-42	na	_	_	_	_	_	_	_	_
 41	em	em	ADP	_	_	43	case	O	_
@@ -9261,9 +9261,9 @@
 19	porção	porção	NOUN	_	Gender=Fem|Number=Sing	15	nmod	O	_
 20	sul	sul	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
 21	,	,	PUNCT	_	_	13	punct	O	_
-22	como	como	ADV	_	_	24	cc	O	_
+22	como	como	SCONJ	_	_	24	mark	O	_
 23	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	24	cop	O	_
-24	possível	possível	ADJ	_	Gender=Masc|Number=Sing	5	conj	O	_
+24	possível	possível	ADJ	_	Gender=Masc|Number=Sing	5	advcl	O	_
 25	visualizar	visualizar	VERB	_	VerbForm=Inf	24	csubj	O	_
 26-27	na	_	_	_	_	_	_	_	_
 26	em	em	ADP	_	_	28	case	O	_
@@ -51349,7 +51349,7 @@
 10	basal	basal	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
 11	fixa	fixo	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
 12	,	,	PUNCT	_	_	15	punct	O	_
-13	como	como	ADV	_	_	15	cc	O	_
+13	como	como	SCONJ	_	_	15	mark	O	_
 14	o	o	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	15	nsubj	O	_
 15	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	advcl	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
@@ -72440,8 +72440,8 @@
 15	grande	grande	ADJ	_	Gender=Masc|Number=Sing	16	amod	O	_
 16	lago	lago	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
 17	,	,	PUNCT	_	_	19	punct	O	_
-18	como	como	ADV	_	_	19	cc	O	_
-19	proposto	propor	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	16	acl	O	_
+18	como	como	SCONJ	_	_	19	mark	O	_
+19	proposto	propor	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	8	advcl	O	_
 20	por	por	ADP	_	_	21	case	O	_
 21	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	19	obl:agent	O	_
 22	e	e	CCONJ	_	_	21	flat:name	O	_
@@ -78178,7 +78178,7 @@
 30	de	de	ADP	_	_	32	case	O	_
 31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	32	det	O	_
 32	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	29	nmod	O	_
-33	estufa	estufo	ADJ	_	Gender=Fem|Number=Sing	32	amod	B=LOCAL:GEO	_
+33	estufa	estufo	NOUN	_	Gender=Fem|Number=Sing	32	compound	B=LOCAL:GEO	_
 34	e	e	CCONJ	_	_	40	cc	O	_
 35	,	,	PUNCT	_	_	36	punct	O	_
 36	consequentemente	consequentemente	ADV	_	_	40	advmod	O	_
@@ -78359,7 +78359,7 @@
 12	de	de	ADP	_	_	14	case	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
 14	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
-15	estufa	estufo	ADJ	_	Gender=Masc|Number=Sing	14	amod	B=LOCAL:GEO	_
+15	estufa	estufo	NOUN	_	Gender=Masc|Number=Sing	14	compound	B=LOCAL:GEO	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	produzido	produzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	10	acl	O	_
 18-19	pela	_	_	_	_	_	_	_	_
@@ -85361,7 +85361,7 @@
 31	gases	gase	NOUN	_	Gender=Masc|Number=Plur	29	nmod	O	_
 32	de	de	ADP	_	_	33	case	O	_
 33	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	31	nmod	O	_
-34	estufa	estufo	ADJ	_	Gender=Fem|Number=Sing	33	amod	B=LOCAL:GEO	_
+34	estufa	estufo	NOUN	_	Gender=Fem|Number=Sing	33	compound	B=LOCAL:GEO	_
 35	..	..	PUNCT	_	_	14	punct	O	_
 
 # text = É a partir desses valores que será calculado a quantidade capturada de gás carbônico..
@@ -88279,7 +88279,7 @@
 40	gases	gase	NOUN	_	Gender=Masc|Number=Plur	38	nmod	O	_
 41	de	de	ADP	_	_	42	case	O	_
 42	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	40	nmod	O	_
-43	estufa	estufo	ADJ	_	Gender=Fem|Number=Sing	42	amod	B=LOCAL:GEO	_
+43	estufa	estufo	NOUN	_	Gender=Fem|Number=Sing	42	compound	B=LOCAL:GEO	_
 44	.	.	PUNCT	_	_	5	punct	O	_
 45	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -103588,7 +103588,7 @@
 10	gases	gase	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
-13	estufa	estufa	NOUN	_	Gender=Fem|Number=Sing	12	nmod	B=LOCAL:GEO	_
+13	estufa	estufa	NOUN	_	Gender=Fem|Number=Sing	12	compound	B=LOCAL:GEO	_
 14	monitoradas	monitorar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	12	acl	O	_
 15	ocorreram	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	3	conj	O	_
 16	como	como	ADP	_	_	17	case	O	_
@@ -104349,7 +104349,7 @@
 7	gases	gase	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
-10	estufa	estufo	ADJ	_	Gender=Fem|Number=Sing	9	amod	B=LOCAL:GEO	_
+10	estufa	estufo	NOUN	_	Gender=Fem|Number=Sing	9	compound	B=LOCAL:GEO	_
 11	,	,	PUNCT	_	_	14	punct	O	_
 12	para	para	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
@@ -104440,7 +104440,7 @@
 17	gases	gase	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	17	nmod	O	_
-20	estufa	estufo	ADJ	_	Gender=Masc|Number=Sing	19	amod	B=LOCAL:GEO	_
+20	estufa	estufo	NOUN	_	Gender=Masc|Number=Sing	19	compound	B=LOCAL:GEO	_
 21	caso	caso	SCONJ	_	_	23	mark	O	_
 22	não	não	ADV	_	Polarity=Neg	23	advmod	O	_
 23	houvesse	haver	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	11	advcl	O	_
@@ -106439,7 +106439,7 @@
 25	gases	gase	NOUN	_	Gender=Masc|Number=Plur	23	nmod	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	25	nmod	O	_
-28	estufa	estufo	ADJ	_	Gender=Fem|Number=Sing	27	amod	B=LOCAL:GEO	_
+28	estufa	estufo	NOUN	_	Gender=Fem|Number=Sing	27	compound	B=LOCAL:GEO	_
 29	..	..	PUNCT	_	_	10	punct	O	_
 
 # text = 8. - Resultados
@@ -108281,7 +108281,7 @@
 50	gases	gase	NOUN	_	Gender=Masc|Number=Plur	48	nmod	O	_
 51	de	de	ADP	_	_	52	case	O	_
 52	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	50	nmod	O	_
-53	estufa	estufo	ADJ	_	Gender=Fem|Number=Sing	52	amod	B=LOCAL:GEO	_
+53	estufa	estufo	NOUN	_	Gender=Fem|Number=Sing	52	compound	B=LOCAL:GEO	_
 54	.	.	PUNCT	_	_	55	punct	O	_
 55	Visto	ver	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	45	acl	O	_
 56	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	55	obj	O	_
@@ -183985,8 +183985,8 @@
 18	produzir	produzir	VERB	_	VerbForm=Inf	7	advcl	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	20	det	O	_
 20	biodiesel	biodiesel	NOUN	_	Gender=Masc|Number=Sing	18	obj	B=PRODUTO	_
-21	como	como	ADP	_	_	22	case	O	_
-22	representado	representar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	18	xcomp	O	_
+21	como	como	SCONJ	_	_	22	mark	O	_
+22	representado	representar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	18	advcl	O	_
 23-24	na	_	_	_	_	_	_	_	_
 23	em	em	ADP	_	_	25	case	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
@@ -184232,8 +184232,8 @@
 26	cana-de-açúcar	cana-de-açúcar	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
 27	por	por	ADP	_	_	28	case	O	_
 28	leveduras	levedura	NOUN	_	Gender=Fem|Number=Plur	23	nmod	O	_
-29	como	como	ADP	_	_	30	case	O	_
-30	apresentado	apresentar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	28	acl	O	_
+29	como	como	SCONJ	_	_	30	mark	O	_
+30	apresentado	apresentar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	6	advcl	O	_
 31	por	por	ADP	_	_	32	case	O	_
 32	Paulillo	Paulillo	PROPN	_	Gender=Masc|Number=Sing	30	obl:agent	O	_
 33	et	et	PROPN	_	Number=Sing	32	flat:name	O	_
@@ -185895,8 +185895,8 @@
 33	de	de	ADP	_	_	35	case	O	_
 34	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	35	det	O	_
 35	transesterificação	transesterificação	NOUN	_	Gender=Fem|Number=Sing	32	obl	B=QUÍMICA	_
-36	como	como	ADP	_	_	37	case	O	_
-37	citado	citar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	35	acl	O	_
+36	como	como	SCONJ	_	_	37	mark	O	_
+37	citado	citar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	14	advcl	O	_
 38	anteriormente	anteriormente	ADV	_	_	37	advmod	O	_
 39	.	.	PUNCT	_	_	14	punct	O	_
 40	.	.	PUNCT	_	_	14	punct	O	_
@@ -219942,8 +219942,8 @@
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	zeólita	zeólita	NOUN	_	Gender=Fem|Number=Sing	8	obj	B=LITOLOGIA	_
 11	Y	Y	PROPN	_	Gender=Fem|Number=Sing	10	nmod	O	_
-12	como	como	ADP	_	_	13	case	O	_
-13	mostrado	mostrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	xcomp	O	_
+12	como	como	SCONJ	_	_	13	mark	O	_
+13	mostrado	mostrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	advcl	O	_
 14-15	na	_	_	_	_	_	_	_	_
 14	em	em	ADP	_	_	16	case	O	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
@@ -241363,8 +241363,8 @@
 32	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	33	det	O	_
 33	calhas	calha	NOUN	_	Gender=Fem|Number=Plur	30	nmod	O	_
 34	cenozóicas	cenozóico	ADJ	_	Gender=Fem|Number=Plur	33	amod	B=GEOCRONOLOGIA	_
-35	como	como	ADV	_	_	36	cc	O	_
-36	mostram	mostrar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	18	conj	O	_
+35	como	como	SCONJ	_	_	36	mark	O	_
+36	mostram	mostrar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	18	advcl	O	_
 37	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	38	det	O	_
 38	áreas	área	NOUN	_	Gender=Fem|Number=Plur	36	obj	O	_
 39	de	de	ADP	_	_	40	case	O	_
@@ -250551,9 +250551,9 @@
 12	verdes	verdes	ADJ	_	Gender=Fem|Number=Plur	11	amod	O	_
 13	)	)	PUNCT	_	_	11	punct	O	_
 14	,	,	PUNCT	_	_	17	punct	O	_
-15	como	como	ADP	_	_	17	case	O	_
+15	como	como	SCONJ	_	_	17	mark	O	_
 16	já	já	ADV	_	_	17	advmod	O	_
-17	esperado	esperar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	acl	O	_
+17	esperado	esperar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	19	advcl	O	_
 18	,	,	PUNCT	_	_	17	punct	O	_
 19	fornece	fornecer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 20	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	19	obj	O	_
@@ -260272,9 +260272,9 @@
 39	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	40	det	O	_
 40	pacote	pacote	NOUN	_	Gender=Masc|Number=Sing	37	nmod	O	_
 41	sedimentar	sedimentar	ADJ	_	Gender=Masc|Number=Sing	40	amod	O	_
-42	como	como	ADV	_	_	44	cc	O	_
+42	como	como	SCONJ	_	_	44	mark	O	_
 43	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	44	aux:pass	O	_
-44	sugerido	sugerir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	28	conj	O	_
+44	sugerido	sugerir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	28	advcl	O	_
 45-46	pela	_	_	_	_	_	_	_	_
 45	por	por	ADP	_	_	47	case	O	_
 46	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	47	det	O	_
@@ -270141,8 +270141,8 @@
 20	300	300	NUM	_	NumType=Card	21	nummod	O	_
 21	metros	metro	NOUN	_	Gender=Masc|Number=Plur	18	obl	O	_
 22	,	,	PUNCT	_	_	24	punct	O	_
-23	como	como	ADV	_	_	24	cc	O	_
-24	sugere	sugerir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	14	conj	O	_
+23	como	como	SCONJ	_	_	24	mark	O	_
+24	sugere	sugerir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	14	advcl	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	interpretação	interpretação	NOUN	_	Gender=Fem|Number=Sing	24	obj	O	_
 27-28	da	_	_	_	_	_	_	_	_
@@ -332496,7 +332496,7 @@
 39	em	em	ADP	_	_	41	case	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
 41	meio	meio	NOUN	_	Gender=Masc|Number=Sing	38	nmod	O	_
-42	ambiente	ambiente	ADJ	_	Gender=Masc|Number=Sing	41	amod	O	_
+42	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	41	compound	O	_
 43	.	.	PUNCT	_	_	14	punct	O	_
 44	.	.	PUNCT	_	_	14	punct	O	_
 
